### PR TITLE
Fixup documentation and naming errors in the 2.0 version

### PR DESCRIPTION
**Pull request by Thibaut-Gudin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # ocaml-cordova-plugin-firebase
 
-Binding to [cordova-plugin-firebase](https://github.com/arnesson/cordova-plugin-firebase)
+Binding to
+[cordova-plugin-firebase](https://github.com/arnesson/cordova-plugin-firebase),
```
Comment (ilankri) [inline README.md:4]: This OCaml library is not a binding to this Cordova plugin, isn't it?

**Review by ilankri — COMMENTED** [review]
Could you please split your changes into several commits doing one single thing?

File: README.md (hunk 1 above)
Comment (ilankri) [reply README.md:4]: OK, after reading https://github.com/besport/ocaml-cordova-plugin-firebase/issues/4#issuecomment-901900133, I see why you write this.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+and more specificaly
```
Comment (ilankri) [inline README.md]:
```suggestion
and more specifically
```

File: README.md (hunk 1 above)
Comment (ilankri) [reply README.md:4]: However, if I am not mistaken, there are other links in the README file that need to be updated.

**Comment by Thibaut-Gudin** [comment]
I see what append, I thought this was the correct API to use for the "Firebase" plugin: "https://github.com/arnesson/cordova-plugin-firebase/blob/master/docs/API.md", but it seems that the function I saw here wasn't available after all.

**Comment by Thibaut-Gudin** [comment]
All the links references should be good now, at least for the direct quotes.

File: src/cordova_firebase.mli [hunk 3]
```diff
@@ -32,23 +32,20 @@ val log_event : event:string ->
 val set_enabled : bool -> unit
   [@@js.global "cordova.plugins.firebase.analytics.setEnabled"]
 
-val get_app_instance_id : unit -> string
-  [@@js.global "cordova.plugins.firebase.analytics.getAppInstanceId"]
-
 [@@@js.stop]
 
-val available : unit -> bool
+val firebase_available : unit -> bool
```
Comment (ilankri) [inline src/cordova_firebase.mli]: Maybe we could simply keep the name without the prefix `firebase`.

Comment (Thibaut-Gudin) [reply src/cordova_firebase.mli]: I wanted to avoid confusion but it's probly juste repetition

**Review by ilankri — APPROVED** [review]

**Pull request merged**